deprecated/bookmark+-1.el

```diff
@@ -7081,7 +7081,7 @@ Return the copy.
 Do not sort if `bmkp-sort-comparer' is nil.
 This is a non-destructive operation: ALIST is not modified.
 
-Sorting is done using using `bmkp-sort-comparer'.
+Sorting is done using `bmkp-sort-comparer'.
```

Comment:
I've only recently added looking for doubled words. For this repository, in a number of cases I didn't simply drop a double, although I did here. I haven't had enough time to get familiar w/ what the right corrections are for doubled words (I've been doing basic spelling fixes for decades...)